docs/pages/configuration/pipelines/functions/run_dependency_pipelines.mdx

```diff
@@ -2,3 +2,37 @@
 title: Function run_dependency_pipelines
 sidebar_label: run_dependency_pipelines
 ---
+
+Runs dependencies for pipelines.
+
+## Arguments
+
+### `--pipeline`
+
+The pipeline to be deployed from the dependency.
+
+### `--exclude`
+
+The dependencies to be excluded from the pipeline.
```

Comment:
Only has an effect if `--all` is used, otherwise this makes no real sense

Reply:
fixed